### PR TITLE
Parity Support

### PR DESCRIPTION
**Pull request by tagomaru**
This PR is for parity support.
It needs to change the repos `priv-testnet`, `orchestration`, `relay`, and `contractor` as well.
The rest will be changed soon.

Please NOTE this is still WIP.

### CODE REVIEW

File: src/polyswarmd/config.py [hunk 1]
```diff
@@ -32,9 +32,15 @@
 }
 
 
-def is_service_reachable(session, uri):
-    r = session.get(uri)
-
+def is_service_reachable(session, uri, is_ethereum=False):
+    if is_ethereum:
```
Comment (myhndl) [inline src/polyswarmd/config.py]: `is_ethereum` is True when checking whether the Ethereum service (either Geth or Parity) is available, correct?

If so, would this break using Geth or does Geth also support the same `POST` message that Parity does?

If they both support the same `POST` message, it's probably best to standardize on the `POST` message, and this should be merged into mainline.

Comment (tagomaru) [reply src/polyswarmd/config.py]: It does support not only parity but also geth since geth can accept POST as well. So I think this can be merged soon.

**Comment by tagomaru** [comment]
@Rizato 
Awesome and Thx!
I fixed it.

File: src/polyswarmd/config.py (hunk 1 above)
Comment (Rizato) [reply src/polyswarmd/config.py]: Just tested this with get. Able to get status correctly.

**Comment by myhndl** [comment]
@Rizato tests failing with contract version mismatch: https://gitlab.polyswarm.io/externalci/polyswarmd/-/jobs/34969

**Pull request merged**